src/pykeen/stoppers/early_stopping.py

```diff
@@ -17,52 +15,45 @@
 from ..utils import fix_dataclass_init_docs
 
 __all__ = [
-    'smaller_than_any_buffer_element',
-    'larger_than_any_buffer_element',
+    'is_improvement',
     'EarlyStopper',
     'StopperCallback',
 ]
 
 logger = logging.getLogger(__name__)
 
+StopperCallback = Callable[[Stopper, Union[int, float], int], None]
 
-def smaller_than_any_buffer_element(buffer: numpy.ndarray, result: float, delta: float = 0.) -> bool:
-    """Decide if a result is better than at least one buffer element, where smaller is better.
-
-    :param buffer:
-        The last results to compare against (excluding the current result).
-    :param result:
-        The current result.
-    :param delta:
-        The minimum improvement.
 
-    :return:
-        Whether the result is at least delta better than at least one value in the buffer.
+def is_improvement(
+    best_value: float,
+    current_value: float,
+    larger_is_better: bool,
+    absolute_delta: float = 0.0,
```

Comment:
Is `absolute_delta` defined somewehere else, too?

Reply:
No, since I changed the default from using absolute delta for significant improvement to relative delta. I was not sure whether we should expose both via the EarlyStopper. Any suggestions for that? Otherwise, we can remove this parameter.

Reply:
For keeping the `EarlyStopper` simple, I think it is better to decide for one of those.

Reply:
I removed the `absolute_delta` variant 
d5742d2

Reply:
Thanks!